<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>331</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="centralLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="titre">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Webdings</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>PluriNotes</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="partiePrincipale">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>60</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Créer une nouvelle note :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonArticle">
         <property name="text">
          <string>Article</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonTask">
         <property name="text">
          <string>Tâche</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonImage">
         <property name="text">
          <string>Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonVideo">
         <property name="text">
          <string>Vidéo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAudio">
         <property name="text">
          <string>Audio</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>150</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>289</y>
      <width>211</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="listRelationLayout">
     <item>
      <widget class="QLabel" name="relationLabel">
       <property name="text">
        <string>Relations :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listRelation"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>261</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="partieGauche">
     <item>
      <widget class="QLabel" name="notesLabel">
       <property name="text">
        <string>Notes :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidgetNotesActives"/>
     </item>
     <item>
      <widget class="QLabel" name="taskLabel">
       <property name="text">
        <string>Tâches : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidgetTasksActives"/>
     </item>
     <item>
      <widget class="QLabel" name="archivesLabel">
       <property name="text">
        <string>Notes archivées :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidgetArchive"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNote">
    <property name="title">
     <string>Notes</string>
    </property>
    <widget class="QMenu" name="menunouvelle_note">
     <property name="title">
      <string>nouvelle note</string>
     </property>
     <addaction name="actionNote"/>
     <addaction name="actionTache"/>
     <addaction name="actionImage"/>
     <addaction name="actionAudio"/>
     <addaction name="actionVideo"/>
    </widget>
    <addaction name="menunouvelle_note"/>
   </widget>
   <widget class="QMenu" name="menuRelations">
    <property name="title">
     <string>Relations</string>
    </property>
    <addaction name="nouvelleRelationAction"/>
   </widget>
   <widget class="QMenu" name="menuCorbeille">
    <property name="title">
     <string>Corbeille</string>
    </property>
    <addaction name="actionafficher"/>
   </widget>
   <widget class="QMenu" name="menuparam_tres">
    <property name="title">
     <string>paramètres</string>
    </property>
    <addaction name="actioncorbeille"/>
   </widget>
   <widget class="QMenu" name="menuApp">
    <property name="title">
     <string>App</string>
    </property>
    <addaction name="actionclose"/>
   </widget>
   <addaction name="menuNote"/>
   <addaction name="menuRelations"/>
   <addaction name="menuCorbeille"/>
   <addaction name="menuparam_tres"/>
   <addaction name="menuApp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNote">
   <property name="text">
    <string>Article</string>
   </property>
  </action>
  <action name="actionTache">
   <property name="text">
    <string>Tache</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionAudio">
   <property name="text">
    <string>Audio</string>
   </property>
  </action>
  <action name="actionVideo">
   <property name="text">
    <string>Video</string>
   </property>
  </action>
  <action name="nouvelleRelationAction">
   <property name="text">
    <string>nouvelle relation</string>
   </property>
  </action>
  <action name="actionafficher">
   <property name="text">
    <string>afficher</string>
   </property>
  </action>
  <action name="actioncorbeille">
   <property name="text">
    <string>corbeille</string>
   </property>
  </action>
  <action name="actionde_merde">
   <property name="text">
    <string>de merde</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
